defaults/main.yml

```diff
@@ -90,7 +90,10 @@ nextcloud_preview_app_jpeg_quality: "60"
 # - `nextcloud_container_image_customizations_dockerfile_body_custom`
 # - `nextcloud_container_image_customized`
 # - `nextcloud_container_image_final`
-nextcloud_container_image_customizations_enabled: "{{ nextcloud_container_image_customizations_php_imageick_installation_enabled }}"
+nextcloud_container_image_customizations_enabled: >
+  "{{ nextcloud_container_image_customizations_php_imageick_installation_enabled 
+  or
+  nextcloud_container_image_customizations_smb_enabled }}"
```

Comment:
The `nextcloud_container_image_customizations_smb_enabled` variable is not defined in `defaults/main.yml`.

It should be defined somewhere below with a default value (`false`).

Also, I suggest not shortening samba to smb. Keeping it as samba, it's more clear what you mean and it allows for grepping.

Reply:
got it. changed the variable name -- seems some people use smb more, but hopefully will always have comment that uses both ways of referring.

also -- was a bit unsure about best practice for variable declarations/definitions. when you said "below", did you mean below in the same file is important? in the end, i imitated the imagemagick declaration, which is earlier in that file.